packages/blockchain-wallet-v4-frontend/src/components/Form/PhoneNumberBox/index.js

```diff
@@ -91,8 +92,6 @@ class PhoneNumberBox extends React.Component {
           defaultCountry={countryCode}
           preferredCountries={['us', 'gb']}
           css={['intl-tel-input', 'form-control']}
-          utilsScript='libphonenumber.js'
-          placeholder='555-555-5555'
```

Comment:
Removing the placeholder makes the component generate placeholders according to the selected country